src/adapters/scene_direction.js

```diff
@@ -28,12 +28,15 @@ const createRouteLabel = (route, vehicle, { lngLat, anchor }) => {
   const element = document.createElement('div');
   element.innerHTML = `
     <div class="routeLabel-vehicleIcon ${getVehicleIcon(vehicle)}"></div>
-    <div>
+    <div class="${vehicle === 'publicTransport' ? 'routeLabel--public-transport' : ''}">
```

Comment:
Why not simply use the existing `if` below which already sets a special class on the element and adapt the selector?

Reply:
huh, I was dumb :D indeed